### DISCUSSION TITLE
Localization of Firefox DevTools User Docs

### DISCUSSION THREAD

**Discussion by cw118** [mdn/mdn-community#600] [Content] [answered]
Originally asked by @myshov in https://github.com/mdn/translated-content/issues/4822#issuecomment-1092661167: after the Tools section was removed from mdn/content in #14140 (removal in translated-content is in progress), are there any plans for localizing the Devtools user docs?

I'm not sure if this is the right place for this post and I apologize if it's not.

**Comment by hamishwillee** [comment, marked as answer]
I don't know. It isn't really a question for MDN now the docs aren't on MDN.

I would ask at the links provided here: https://firefox-source-docs.mozilla.org/setup/contributing_code.html#how-to-contribute-code-to-firefox

**Reply by cw118** [reply]
@hamishwillee Thank you for taking the time to reply and for the link! I had searched all over the Devtools docs but somehow missed that page :/

**Reply by hamishwillee** [reply]
No worries! It did feel a bit hidden :-(

**Comment by pepelsbey** [comment]
Closing as answered/settled.
